### ISSUE TITLE
The password strength tool seems to be a bit silly

### ISSUE THREAD

**Issue by zoredache** [cronicle-edge/cronicle-edge#157]
I got a warning that my password was 'insecure'.  Which is silly since I am using a 15 word diceware password.  `diceware -d '-' -n 15`. So a password like:

    Tannery-Skinning-Violet-Grain-Negate-Agreed-Agreed-Resonate-Deviancy-Doctrine-Smudge-Crawlers-Trustable-Floss-Upstairs

Current NIST guidelines strong advice that the silly complexity rules many tools try to use is mostly a waste of time.  Length is basically everything.

Not sure if you implement your password strength checking yourself, or if it came as part of a library, but it would be nice if it was updated.

P.S. This isn't very important, and me just nit-picking.

**Comment by mikeTWC1984** [comment]
Yeah, it's just silly password checker. Original cronicle using some heavy library which is 1MB in size, I just removed it and put some dummy analyzer for backwards compatibility. I'll adjust it a bit on the next release
